config_test.go

```diff
@@ -58,6 +58,28 @@ func TestParseWrongDSN(t *testing.T) {
 	}
 }
 
+func TestParseWrongDNSParams(t *testing.T) {
+	case1 := map[string][]string{
+		"enable_http_compression": {"foo"},
+	}
+	case2 := map[string][]string{
+		"timeout": {"50"},
+	}
+	case3 := map[string][]string{
+		"database": {"whatever"},
+	}
+
+	testCases := []map[string][]string{
```

Comment:
IMHO, it would be better to do this without additional variables,
```
testCases := []map[string][]string{
    {"enable_http_compression": {"50"}},
    {"timeout": {"50"}},
    {"database": {"whatever"}},
}
```

Reply:
addressed.